torch/csrc/dynamo/eval_frame.c

```diff
@@ -59,6 +59,24 @@ static PyObject* THPPyInterpreterFrame_f_lasti(THPPyInterpreterFrame* self, PyOb
   return PyLong_FromLong(_PyInterpreterFrame_LASTI(self->frame));
 }
 
+static PyObject* THPPyInterpreterFrame_f_lineno(THPPyInterpreterFrame* self, PyObject* _noargs) {
+  if (!self->frame->frame_obj) {
```

Comment:
Can attempt to create `frame_obj` here using `THP_PyFrame_GetFrameObject` in `torch/csrc/dynamo/cpython_defs.c`.

Reply:
The new code... doesn't do this?

Reply:
But like, `f_lineno` is not the same thing as `co_firstlineno`???